README.md

```diff
@@ -9,14 +9,16 @@ install that simulates Ironic multi-tenancy through the use of `project_owner_id
 
 ### Installation
 
-To  install the python code:
+To  install the python code without PyPI:
 
 ```
     $ git clone https://github.com/CCI-MOC/esi-leap
     $ cd esi-leap
     $ sudo python setup.py install
 ```
 
+To install the package using PyPI:
```

Comment:
I'd put this before the instructions for installing from source; it's what we want people to do. Also, maybe reword as "To install as a package"